lib/src/components/containers/newMember.dart

```diff
@@ -88,9 +106,13 @@ class NewMember extends PComponent<NewMemberProps> {
                                     ..className = 'control'
                                     ..children = [
                                       new VInputElement()
-                                        ..className = 'input'
+                                        ..className = 'input ${state.firstNameIsValid ? '' : 'is-danger'}'
+                                        ..onInput = _firstNameValidation
                                         ..id = 'fName-input'
-                                        ..placeholder = "First Name"
+                                        ..placeholder = "First Name",
+                                      new VParagraphElement()
+                                        ..className = 'help is-danger ${state.firstNameIsValid ? 'is-invisible' : ''}'
+                                        ..text = 'First name is invalid'
```

Comment:
I vote we do 'First name is Required' instead, since that would make a little more sense from a user perspective.